packages/server/lib/controllers/v1/team/putTeam.ts

```diff
@@ -29,11 +31,21 @@ export const putTeam = asyncWrapper<PutTeam>(async (req, res) => {
         return;
     }
 
-    const { account } = res.locals;
+    const { account, plan } = res.locals;
     const body: PutTeam['Body'] = val.data;
 
     await accountService.editAccount({ id: account.id, ...body });
 
+    if (plan?.stripe_customer_id && plan?.orb_customer_id) {
+        const stripe = getStripe();
+        await Promise.all([
+            stripe.customers.update(plan.stripe_customer_id, {
+                name: body.name
+            }),
+            billing.updateCustomer(plan.orb_customer_id, body.name)
```

Comment:
can this lead to one being updated an not the other? Is that a problem?
also how it any error handled?

Reply:
not really a problem it's just for display